WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -1572,6 +1572,20 @@ public LoadTagsTask(FilteredRecyclerView.FilterCriteriaAsyncLoaderListener liste
         @Override
         protected ReaderTagList doInBackground(Void... voids) {
             ReaderTagList tagList = ReaderTagTable.getDefaultTags();
+
+            // translate default tags
+            // ref: https://github.com/wordpress-mobile/WordPress-Android/issues/5240
+            Context context = getContext();
```

Comment:
This seems fragile since it's changing the title in a single place for a single purpose. It may work better to do this in `ReaderTag.getLabel()`, since the purpose of that method is to return the tag name for display. Example:

```
@Override
public String getLabel() {
    if (tagType == ReaderTagType.DEFAULT) {
        // translate default tags
        // ref: https://github.com/wordpress-mobile/WordPress-Android/issues/5240
        if (isDiscover()) {
            return WordPress.getContext().getString(R.string.reader_discover_default_tag);
        } else if (isFollowedSites()) {
            return WordPress.getContext().getString(R.string.reader_followed_default_tag);
        } else if (isPostsILike()) {
            return WordPress.getContext().getString(R.string.reader_liked_default_tag);
        } else {
            return getTagTitle();
        }
    } else if (isTagDisplayNameAlphaNumeric()) {
        return getTagDisplayName().toLowerCase();
    } else if (hasTagTitle()) {
        return getTagTitle();
    } else {
        return getTagDisplayName();
    }
}
```